Source/Editor/Windows/MainWindow.xaml.cs

```diff
@@ -361,13 +361,13 @@ private void OpenElementWindow(ModelBase model, string elementName, DescriptorBa
             LoadFromRam();
         }
 
-        private void CopyAppenderToClipboard(object sender, RoutedEventArgs e)
+        private void CopyElementToClipboard(object sender, RoutedEventArgs e)
         {
-            AppenderModel appender = (AppenderModel)((Button)sender).DataContext;
+            ModelBase model = (ModelBase)((Button)sender).DataContext;
 
-            Clipboard.SetText(appender.Node.OuterXml);
+            Clipboard.SetText(model.Node.OuterXml);
```

Comment:
Node is allowed to be null - but it won't be in this context, even if it's a new, unsaved element.